.taskcluster.yml

```diff
@@ -2,123 +2,237 @@
+              - /bin/bash
+              - --login
+              - -cx
+              # The rest of the command must be defined below
+            env:
+              GIT_URL: ${clone_url}
+              GIT_REF: ${head_branch}
+              GIT_SHA: ${head_rev}
+              TASK_FOR: ${tasks_for}
+              TASK_OWNER: *task_owner
+              TASK_SOURCE: *task_source
+            features:
+              taskclusterProxy: true
+      in:
+        $if: 'tasks_for in ["github-pull-request", "github-push"]'
+        then:
+          $let:
+            pr_or_push_parameters:
+              payload:
+                command:
+                  - >-
+                    git init repo &&
+                    cd repo &&
+                    git fetch --depth 1 ${clone_url} ${head_branch} &&
+                    git reset --hard ${head_rev} &&
+                    python3 automation/taskcluster/decision_task.py pr-or-push
+          in:
+            - $if: 'tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "edited", "synchronize"]'
+              then:
+                $let:
+                  pull_request_title: ${event.pull_request.title}
+                  pull_request_number: ${event.pull_request.number}
+                  pull_request_url: ${event.pull_request.html_url}
+                in:
+                  $mergeDeep:
+                    - {$eval: 'default_task_definition'}
+                    - {$eval: 'pr_or_push_parameters'}
+                    - payload:
+                        env:
+                          GITHUB_PULL_TITLE: ${pull_request_title}
+                      metadata:
+                        name: 'Application Services - Decision task (Pull Request #${pull_request_number})'
+                        description: 'Building and testing Application Services - triggered by [#${pull_request_number}](${pull_request_url})'
+            - $if: 'tasks_for == "github-push" && head_branch == "refs/heads/master"'
+              then:
+                $mergeDeep:
+                  - {$eval: 'default_task_definition'}
+                  - {$eval: 'pr_or_push_parameters'}
+                  - metadata:
+                      name: Application Services - Decision task
+                      description: Schedules the build and test tasks for Application Services.
+        else:
+          $if: 'tasks_for in ["github-release", "cron"]'
+          then:
+            $let:
+              command_staging_flag:
+                $if: 'is_repo_trusted'
+                then: ''
+                else: '--staging'
 
-        payload:
-          maxRunTime: {$eval: '20 * 60'}
-          # https://github.com/servo/taskcluster-bootstrap-docker-images#decision-task
-          image: "servobrowser/taskcluster-bootstrap:decision-task@sha256:28045b7ec0485ef363f8cb14f194008b47e9ede99f2ea40a1e945e921fce976e"
-          features:
-            taskclusterProxy: true
+              # beetmover_worker_type:
```

Comment:
What does this commented out code do, and why is it here?

Reply:
As discussed on Slack I will un-comment this code in https://github.com/mozilla/application-services/pull/744